counter/src/main/java/org/infinispan/counter/configuration/Reliability.java

```diff
@@ -0,0 +1,23 @@
+package org.infinispan.counter.configuration;
+
+/**
+ * Reliability mode available for {@link org.infinispan.counter.api.CounterManager}.
+ *
+ * @author Pedro Ruivo
+ * @since 9.0
+ */
+public enum Reliability {
+   /**
+    * If the cluster splits in multiple partition, all of them are allowed to continue read/update the counter.
+    * <p>
+    * When the cluster merges back, one partition's updates are kept the others partition's updates are lost.
+    */
+   AVAILABLE,
+   /**
+    * If the cluster splits in multiple partitions, one of the is allowed to read/update the counter and the others are
+    * only allowed to read it.
+    * <p>
+    * When the cluster merges back, the updated counter's value is sent back.
+    */
+   CONSISTENT
```

Comment:
For some reason, I though the minory partition was able to read. I'll double check and update the documentation accordingly.